sirepo/util.py

```diff
@@ -183,6 +184,19 @@ def flask_app():
 
     return flask.current_app or None
 
+
+def import_sim_module(sim_type, module):
+    """Import module for sim type
+
+    sim_type will be used to find the root package (ex sirepo)
+    and module will be appended to it (ex sim_data.srw)
+    """
+    import sirepo.feature_config
+    return importlib.import_module(
+        sirepo.feature_config.cfg().dynamic_sim_types.get(sim_type, 'sirepo') + \
```

Comment:
This should just search a path.

Reply:
58607069 is this what you mean by search a path?